venter/admin.py

```diff
@@ -115,9 +115,21 @@ def send_emails(modeladmin, request, queryset):
             # Retrieves a list of email ids for the selected authorities. Excludes the authority bodies with no email id
             recipient_list = queryset.values_list('authorities__email', flat=True).exclude(authorities__email=None)
 
+            authority = item.authorities.values_list('name', flat=True)
+            auth = ', '
+            auth = auth.join(authority)
```

Comment:
Squash 119 and 120 into one line. This is somewhat confusing. Also change the variable name to something more sensible